internal/objects/drop.go

```diff
@@ -0,0 +1,47 @@
+package objects
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/gleanerio/nabu/pkg/config"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+	"io/ioutil"
+	"net/http"
+)
+
+// Drop removes a graph
+func Drop(v1 *viper.Viper, g string) ([]byte, error) {
```

Comment:
Why is there a 'graph' and an 'objects' is this a more 'graphstore'

Reply:
Do you mean why is a "graph" operation in the objects package space?  

It's a good point.  The files drop.go, dropget.go and insert.go seem like they should be in the graph package, not the object package.  Is this what you were highlighting here?

Reply:
Was looking at all the data storing/retrieving procedures thinking object really about accessing the datastores
But object is fine.

Reply:
moved some of the files to more logical packages in the internal section